From 2c84ee54a75770e291f38d4ebb2c31c8f2c3b8c5 Mon Sep 17 00:00:00 2001
From: Lukas Rusak <lorusak@gmail.com>
Date: Wed, 12 Jul 2023 19:58:08 -0700
Subject: [PATCH] PythonBindings: TypeInfo: add initialization of tp_watched
 for PyTypeObject

This member was added in upstream commit https://github.com/python/cpython/commit/82ccbf69a842db25d8117f1c41b47aa5b4ed96ab

This change first appeared in Python v3.12.0a1

Signed-off-by: Lukas Rusak <lorusak@gmail.com>
---
 xbmc/interfaces/python/swig.cpp | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/xbmc/interfaces/python/swig.cpp b/xbmc/interfaces/python/swig.cpp
index f9374d53e6d94..0c49f87ca84e1 100644
--- a/xbmc/interfaces/python/swig.cpp
+++ b/xbmc/interfaces/python/swig.cpp
@@ -71,6 +71,9 @@ namespace PythonBindings
 #endif
 #if PY_VERSION_HEX < 0x03090000
       0,
+#endif
+#if PY_VERSION_HEX >= 0x030C00A1
+      0,
 #endif
     };
 
